### PR TITLE
iOS 15 and rootless support

### PR DESCRIPTION
**Pull request by NyaMisty**
This PR fixes @PoomSmart 's #38 , turning into a linear commit history.

### CODE REVIEW

**Comment by Mabdelwanis** [comment]
it's crash when tap on NSURLSession 
Dopamine IOS 15.4.1

**Comment by NSExceptional** [comment]
@Mabdelwanis Can you post the crash log?

**Comment by Mabdelwanis** [comment]
@NSExceptional 
https://pastebin.com/raw/epYwDfak

**Comment by L1ghtmann** [comment]
Bumping

File: Tweak.xm [hunk 1]
```diff
@@ -149,7 +151,10 @@ inline BOOL flexAlreadyLoaded() {
     self = %orig;
     if ([present isKindOfClass:%c(FLEXNavigationController)]) {
         // Enable half height sheet
-        self._presentsAtStandardHalfHeight = YES;
+        if ([self respondsToSelector:@selector(_presentsAtStandardHalfHeight)])
+            self._presentsAtStandardHalfHeight = YES;
+        else
+            self._detents = @[[%c(_UISheetDetent) _mediumDetent], [%c(_UISheetDetent) _largeDetent]];
```
Comment (NSExceptional) [inline Tweak.xm:157]:
```suggestion
        if ([self respondsToSelector:@selector(_presentsAtStandardHalfHeight)]) {
            self._presentsAtStandardHalfHeight = YES;
        } else {
            self._detents = @[[%c(_UISheetDetent) _mediumDetent], [%c(_UISheetDetent) _largeDetent]];
        }
```

**Review by NSExceptional — APPROVED** [review]

**Pull request merged**